servicex/resources/jwt/user_registration.py

```diff
@@ -53,6 +56,11 @@ def post(self):
 
         try:
             new_user.save_to_db()
+            webhook_url = os.environ.get("SLACK_WEBHOOK_URL", default=None).strip("'")
+            if webhook_url is not None:
+                requests.post(webhook_url, json.dumps({
+                    "text": f"New signup from {new_user.username}"
+                }))
```

Comment:
Check for result and dump something to stdout that we sent the slack message and whether it failed. This will make debugging easier